File: examples/scenarios/streak-protection-escalation/snippets/ios/bootstrap-config.swift
// Example placement: app startup or feature bootstrap (pseudo-code).
// Adapt to: your app architecture, notification integration, and actual OpenClix runtime API names.
// Shared config: examples/scenarios/streak-protection-escalation/openclix.config.json
// Events used: replaceConfig
// Note: OpenClix APIs shown here are placeholders until the reference SDK is published.

import Foundation

func loadstreakProtectionEscalationConfig(openClix: OpenClixManagerLike) throws {
  let configPath = "examples/scenarios/streak-protection-escalation/openclix.config.json"
  let data = try Data(contentsOf: URL(fileURLWithPath: configPath))
  let configJSON = try JSONSerialization.jsonObject(with: data)
  try openClix.replaceConfig(configJSON)
}

File: examples/scenarios/streak-protection-escalation/snippets/android/BootstrapConfig.kt
// Example placement: Application startup or feature bootstrap (pseudo-code).
// Adapt to: your app architecture, notification integration, and actual OpenClix runtime API names.
// Shared config: examples/scenarios/streak-protection-escalation/openclix.config.json
// Events used: replaceConfig
// Note: OpenClix APIs shown here are placeholders until the reference SDK is published.

fun loadStreakProtectionEscalationConfig(openClix: OpenClixManagerLike, configText: String) {
    // In a real app, read from assets/raw resources and parse into your config model.
    val configJson = parseJsonObject(configText)
    openClix.replaceConfig(configJson)
}
